### PR TITLE
[Hub Generated] Review request for Microsoft.ApiManagement to add version stable/2019-12-01

### PR DESCRIPTION
**Pull request by solankisamir**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/solankisamir/azure-rest-api-specs/apim-2019-12-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/resource-manager/readme.md [hunk 1]
```diff
@@ -28,15 +28,56 @@ These are the global settings for the ApiManagement API.
...
-tag: package-preview-2019-12
+tag: package-2019-12
 ```
 
 
+### Tag: package-2019-12
+
+These settings apply only when `--tag=package-2019-12` is specified on the command line.
+
+```yaml $(tag) == 'package-2019-12'
```
Comment (raych1) [inline specification/apimanagement/resource-manager/readme.md]: Add a space before `yaml`.

Comment (solankisamir) [reply specification/apimanagement/resource-manager/readme.md]: @raych1 This entire PR was generated by Open api hub. I didn’t push any commit. Maybe we need to fix something there. I can fix it here.

**Review by raych1 — COMMENTED** [review]
LGTM.

**Review by pilor — APPROVED** [review]

**Review by vfedonkin — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/apimgateways.json [hunk 2]
```diff
@@ -0,0 +1,997 @@
...
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/gateways/{gatewayId}/listKeys": {
+      "post": {
```
Comment (VeryEarly) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/apimgateways.json:355]: why this is a POST not GET?

Comment (vfedonkin) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/apimgateways.json:355]: @VeryEarly , it's an operation to retrieve secrets (keys). Common approach is to use POST operation so RBAC system may allow all GET methods for Readers, when not allowing POST methods.

**Pull request merged**